src/common/summarize-profile.js

```diff
@@ -250,8 +271,10 @@ export function categorizeThreadSamples(profile) {
     });
 
     if (process.env.NODE_ENV === 'development') {
-      const uncategorized = countUncategorizedStacks(profile, summaries);
-      logUncategorizedSamples(uncategorized);
+      const cat = process.env.DUMP_CATEGORY || 'uncategorized';
```

Comment:
Hmm... overloading the env variable feels a bit magical to me, especially without docs explaining how to use it.

nit: Most of the code here uses the full names of things, and aren't abbreviated versions. Stylistically I would prefer making things more verbose to aid in legibility, e.g. `categoryToCount`.